web/src/pages/onboarding.tsx

```diff
@@ -143,6 +143,15 @@ const OnboardingPage = (props: Props): ReactElement => {
     return !!LookupIndustryById(industryId).id;
   };
 
+  const flowQueryParamIsValid = (flow: string | undefined): boolean => {
+    return (
+      //eslint-disable-next-line unicorn/prefer-array-some
+      businessPersonas.find((el) => {
+        return el === flow;
+      }) !== undefined
+    );
+  };
```

Comment:
why are we disabling this linting rule? It seems to be giving you helpful advice about how this code could be improved and simplified.